src/main/java/com/epam/esm/controller/OrderController.java

```diff
@@ -57,16 +53,10 @@ public TinyOrderInfoDTO fetchUserOrder(@RequestParam(name = "user_id", required
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
-    public Page<Order> fetchUserOrders(@RequestParam(name = "user_id", required = true) long userId,
-                                       @RequestParam(name = "page") Optional<Integer> page,
-                                       @RequestParam(name = "pageSize") Optional<Integer> pageSize,
-                                       @RequestParam(name = "sortOrder") Optional<Sort> sortOrder) {
+    public PagedModel<OrderModel> fetchUserOrders(@PathVariable long id, Pageable pageable) {
+        org.springframework.data.domain.Page<Order> userOrders = orderService.getUserOrders(id, pageable);
```

Comment:
Why you've insert here full path of class?

Reply:
fixed, I don't remember how it happened)